composer.json

```diff
@@ -9,7 +9,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.4 | ^8.0.2 | ^8.1 | ^8.2",
```

Comment:
`8.0.2` was added in #225, I can find no justification for the oddly-specific constraint. Possibly it was just the latest version at the time. Anyway, the specificity of it conflicts with the config php definition below.